e2e/cypress/integration/search/post_search_display_spec.js

```diff
@@ -10,24 +10,37 @@
 // Stage: @prod
```

Comment:
Please remove this prod metadata. We'll add it back once we found the tests added here to be stable for production test.
```suggestion
```